x-pack/legacy/plugins/reporting/server/lib/esqueue/worker.js

```diff
@@ -87,7 +87,7 @@ export class Worker extends events.EventEmitter {
     this._client = this.queue.client;
     this.jobtype = type;
     this.workerFn = workerFn;
-    this.checkSize = opts.size || 10;
+    this.checkSize = opts.size || 1;
```

Comment:
Should this even be an option?

Reply:
As far as I can see nothing really utilizes this `size` param, and I'm a +1 on making it static since I'm not sure what the merit is in doing `10`

Reply:
Thanks - I'll add the 1 as a named constant in somewhere

Reply:
effab9eddcc0432e4718334db2e6114cc24a2295

Reply:
Looks like this is still here and now unused?

Reply:
✅ 52a00ce159